### PR TITLE
www-apps/radicale: enable test and drop python3.7

### PR DESCRIPTION
**Pull request by scardracs**
Added required packages for enabling test on pytest
Fixed some typos

Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: www-apps/radicale/radicale-3.0.6-r2.ebuild [hunk 1]
```diff
@@ -23,50 +23,60 @@ RDEPEND="
...
+BDEPEND="test? ( ${RDEPEND} )"
+
 S="${WORKDIR}/${MY_P}"
 
-RDIR=/var/lib/${PN}
+RDIR=/var/lib/"${PN}"
 
-python_install_all() {
-	rm README* || die
+DOCS=( DOCUMENTATION.md NEWS.md README.md )
```
Comment (00-matt) [inline www-apps/radicale/radicale-3.0.6-r2.ebuild:38]: `README.md` doesn't seem to contain any useful info.

```suggestion
DOCS=( DOCUMENTATION.md NEWS.md )
```

File: www-apps/radicale/radicale-3.0.6-r2.ebuild [hunk 2]
```diff
@@ -23,50 +23,60 @@ RDEPEND="
...
 pkg_postinst() {
 	local _erdir="${EROOT}${RDIR}"
 
 	einfo "A sample WSGI script has been put into ${EROOT}/usr/share/${PN}."
 	einfo "You will also find there an example FastCGI script."
 	if [[ $(stat --format="%U:%G:%a" "${_erdir}") != "${PN}:${PN}:750" ]]
 	then
-		ewarn "Unsafe file permissions detected on ${_erdir}. This probably comes"
-		ewarn "from an earlier version of this ebuild."
+		echo ""
```
Comment (00-matt) [inline www-apps/radicale/radicale-3.0.6-r2.ebuild:75]: https://devmanual.gentoo.org/tools-reference/echo/index.html#abuse-of-echo

File: www-apps/radicale/radicale-3.0.6-r2.ebuild (hunk 1 above)
Comment (scardracs) [reply www-apps/radicale/radicale-3.0.6-r2.ebuild]: Fixed

File: www-apps/radicale/radicale-3.0.6-r2.ebuild (hunk 2 above)
Comment (scardracs) [reply www-apps/radicale/radicale-3.0.6-r2.ebuild]: Fixed

**Review by juippis — APPROVED** [review]
Note that fixing test phase and/or adding/removing python-compat doesn't require revbumping, but I also see you added a new RDEPEND.

**Pull request closed without merging**